Tests/RxCocoaTests/UIPickerView+RxTests.swift

```diff
@@ -37,4 +37,137 @@ final class UIPickerViewTests: RxTest {
+                adapterDeallocated = true
+            })
+        }
+        
+        XCTAssertFalse(adapterDeallocated)
+        autoreleasepool { adapterSubscription.dispose() }
+        XCTAssertTrue(adapterDeallocated)
+    }
+    
+    func testReceiveSelectionEventsAfterBindingAdapter() {
+        var selectedItem: (Int, Int)!
+        let pickerView = UIPickerView(frame: CGRect(x: 0, y: 0, width: 1, height: 1))
+        let items = Observable.just([[1, 2], [1, 2, 3]])
+        let adapter = TestPickerViewAdapter()
+        
+        _ = pickerView.rx.itemSelected.subscribe(onNext: { item in
+            selectedItem = item
+        })
+        
+        _ = items.bind(to: pickerView.rx.items(adapter: adapter))
+        
+        pickerView.delegate!.pickerView!(pickerView, didSelectRow: 0, inComponent: 1)
+        
+        XCTAssertEqual(selectedItem.0, 0)
+        XCTAssertEqual(selectedItem.1, 1)
+    }
+    
+    func testDisplayItemsAfterItemSelectionSubscription() {
+        var titleForRowCalled = false
+        let pickerView = UIPickerView(frame: CGRect(x: 0, y: 0, width: 100, height: 50))
+        let items = PublishSubject<[[Int]]>()
+        let stubAdapter = StubPickerViewAdapter()
+        
+        stubAdapter.titleForRow = { _, _, _ in
+            titleForRowCalled = true
+            return ""
+        }
+        _ = items.bind(to: pickerView.rx.items(adapter: stubAdapter))
+        _ = pickerView.rx.itemSelected.subscribe(onNext: { _ in
+        })
+        
+        items.onNext([[1, 2], [1, 2, 3]])
+        
+        XCTAssertTrue(titleForRowCalled)
+    }
+}
+
+final class StubPickerViewAdapter: TestPickerViewAdapter {
+    var titleForRow: ((UIPickerView, Int, Int) -> String?)!
+    
+    override func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
+        return titleForRow(pickerView, row, component)
+    }
+}
+
+class TestPickerViewAdapter: NSObject, RxPickerViewDataSourceType, UIPickerViewDataSource, UIPickerViewDelegate {
+    typealias Element = [[Int]]
+    private var items: [[Int]] = []
+    
+    func numberOfComponents(in pickerView: UIPickerView) -> Int {
+        return items.count
+    }
+    
+    func pickerView(_ pickerView: UIPickerView, numberOfRowsInComponent component: Int) -> Int {
+        return items[component].count
+    }
+    
+    func pickerView(_ pickerView: UIPickerView, titleForRow row: Int, forComponent component: Int) -> String? {
+        return "\(items[component][row])"
+    }
+    
+    func pickerView(_ pickerView: UIPickerView, observedEvent: Event<Element>) {
+        switch observedEvent {
+        case .next(let items):
+            self.items = items
+            pickerView.reloadAllComponents()
+            pickerView.setNeedsLayout()
+            pickerView.layoutIfNeeded() //force layout in test
+        default:
+            break
+        }
+    }
 }
```

Comment:
We are missing `DelegateProxyTests`. Take a look at `DelegateProxyTest+UIKit.swift`.